packages/ember-testing/lib/helpers/current_route_name.js

```diff
@@ -1,3 +1,17 @@
+/*
```

Comment:
Can you move this down to just above `currentRouteName` function below?

Also, the comment needs to start with `/**` for YUIDoc to properly parse it as documentation.